src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ONE_ELEMENT = 1;
+    private static final double SIZE_GROWTH_INDEX = 1.5;
+
+    private Object[] elementData;
+    private int capacity = DEFAULT_CAPACITY;
```

Comment:
you don't need this parameter. If you need to know maximum array size, you can use `elementData.lenght`

Reply:
done